src/main/osd/osd_warnings.c

```diff
@@ -248,6 +249,15 @@ void renderOsdWarning(char *warningText, bool *blinking, uint8_t *displayAttr)
 
 #endif // USE_GPS_RESCUE
 
+#ifdef USE_POS_HOLD_MODE
+    if (osdWarnGetState(OSD_WARNING_POSHOLD_FAILED) && showPosHoldWarning()) {
+        tfp_sprintf(warningText, "POS_HOLD_FAIL");
```

Comment:
```suggestion
        tfp_sprintf(warningText, "POS HOLD FAIL");
```

Reply:
Thanks, wasn't sure if spaces were allowed.  Fixed.

Reply:
> Thanks, wasn't sure if spaces were allowed. Fixed.

I think I should have seen this warning on osd when modes wasn't activated, but I saw only ****